README.md

```diff
@@ -293,20 +293,20 @@ Here we'll describe the settings you can configure on your Bottlerocket instance
 
 You can see the current settings with an API request:
 
-```
+```sh
 apiclient get settings
 ```
 
 This will return all of the current settings in JSON format.
 For example, here's an abbreviated response:
 
-```
+```json
 {"motd":"...", {"kubernetes": ...}}
```

Comment:
This one is also awkward: the three `...` aren't valid JSON so they get highlighted as an error. Maybe it just needs to be wrapped in a string like the `motd` key is?

![Screen Shot 2022-10-07 at 10 28 13 AM](https://user-images.githubusercontent.com/23109390/194602378-2a7b7d21-9766-4f1c-81f7-d133b96db1e5.png)

Reply:
I actually think this may be OK. It kind of highlights "we're giving you a JSON example here, but it's obvious this is just a snippet and not a real valid JSON structure". But I could go either way if there's a strong opinion against that.

Reply:
Sounds good! I don't have crazy strong opinions on this, so whatever you think looks best!

Reply:
I agree that highlighted errors are not ideal.  `txt` or put the quotes if `json`, either way.